### ISSUE TITLE
Warning while quantizing the resnet18_quant example

### ISSUE THREAD

**Issue by amitpant7** [Xilinx/Vitis-AI#1390]
I was trying to quantize resnet 18 from the example provided in github repo but while deploying i facing warnings as 

(vitis-ai-pytorch) vitis-ai-user@c57949817fd4:/workspace/test$ sudo /opt/vitis_ai/conda/envs/vitis-ai-pytorch/bin/python resnet18_quant.py --quant_mode test --subset_len 1 --batch_size=1 --deploy

[VAIQ_NOTE]: Loading NNDCT kernels...
-------- Start resnet18 test 

[VAIQ_NOTE]: OS and CPU information:
               system --- Linux
                 node --- c57949817fd4
              release --- 6.2.0-39-generic
              version --- #40~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Nov 16 10:53:04 UTC 2
              machine --- x86_64
            processor --- x86_64

[VAIQ_NOTE]: Tools version information:
                  GCC --- GCC 9.4.0
               python --- 3.7.12
              pytorch --- 1.12.1
        vai_q_pytorch --- 3.0.0+a44284e+torch1.12.1

[VAIQ_NOTE]: Quant config file is empty, use default quant configuration

[VAIQ_NOTE]: Quantization test process start up...

[VAIQ_NOTE]: =>Quant Module is in 'cpu'.

[VAIQ_NOTE]: =>Parsing ResNet...

[VAIQ_NOTE]: Start to trace and freeze model...

[VAIQ_NOTE]: The input model ResNet is torch.nn.Module.

[VAIQ_NOTE]: Finish tracing.

[VAIQ_NOTE]: Processing ops...
██████████████████████████████████████████████████| 71/71 [00:00<00:00, 3727.76it/s, OpInfo: name = return_0, type = Return]                      

[VAIQ_NOTE]: =>Doing weights equalization...

[VAIQ_NOTE]: =>Quantizable module is generated.(quantize_result/ResNet.py)

[VAIQ_NOTE]: =>Get module with quantization.
Files already downloaded and verified
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  5.01it/s]
loss: 9.38006
top-1 / top-5 accuracy: 0 / 0
/opt/vitis_ai/conda/envs/vitis-ai-pytorch/lib/python3.7/site-packages/pytorch_nndct/nn/modules/prim_ops.py:116: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if not (list(self.node.out_tensors[0].shape[1:]) == list(input.size())[1:]):
/opt/vitis_ai/conda/envs/vitis-ai-pytorch/lib/python3.7/site-packages/pytorch_nndct/quantization/torchquantizer.py:53: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if inf.sum() > 0 or nan.sum() > 0:
/opt/vitis_ai/conda/envs/vitis-ai-pytorch/lib/python3.7/site-packages/pytorch_nndct/nn/modules/fix_ops.py:68: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  tensor.storage().size() != tensor.numel()):
/opt/vitis_ai/conda/envs/vitis-ai-pytorch/lib/python3.7/site-packages/pytorch_nndct/nn/modules/adaptive_avg_pool.py:42: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  input_size = [int(dim) for dim in input.shape[2:]]

[VAIQ_NOTE]: ResNet_int.pt is generated.(quantize_result/ResNet_int.pt)

[VAIQ_NOTE]: ResNet_int.onnx is generated.(quantize_result/ResNet_int.onnx)

[VAIQ_NOTE]: =>Converting to xmodel ...

[VAIQ_NOTE]: =>Successfully convert 'ResNet' to xmodel.(quantize_result/ResNet_int.xmodel)

Does it impact my xmodel or not ? and how to resolve these warnings. I am using vitis ai 3.0
